base/shared/src/main/scala/scalaz/typeclass/EqSyntax.scala

```diff
@@ -1,9 +1,11 @@
 package scalaz
 package typeclass
 
+import scala.language.experimental.macros
+
 trait EqSyntax {
   implicit final class ToEqOps[A: Eq](a: A) {
-    // TODO: macro syntax
-    def ===(b: A): Boolean = implicitly[Eq[A]].equal(a, b)
+    private[typeclass] type Equal
+    def ===(f: A): Boolean = macro meta.SymOps.fa_1[Equal]
```

Comment:
why uncap it? `def ===(f: A): Boolean = macro meta.SymOps.fa_1[equal]` seems fine to me.

I wish we had literal types in scala 2; `macro meta.SymOps.fa_1["equal"]` would be better still.

Reply:
Fixed.

Reply:
Thanks!